packages/invoices/src/OutboundShipment/DetailView/modals/ItemDetails.tsx

```diff
@@ -90,13 +90,15 @@ export const ItemDetails: React.FC<ItemDetailsProps> = ({ item, onSubmit }) => {
     value: Item | null
   ) => {
     setSelectedItem(value);
-    setValue('id', value?.id || '');
-    setValue('code', value?.code || '');
-    setValue('name', value?.name || '');
-    trigger('id');
+    setValue('itemId', value?.id || '');
```

Comment:
yeah, this is a bit hokey. just a temporary thing until the FEFO logic comes in.